appserviceenvironment-serviceplan-logicapp/azuredeploy.json

```diff
@@ -0,0 +1,262 @@
+﻿{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "metadata": {
```

Comment:
Needs allowed values for the regions Core RPs exist in today. See https://github.com/Azure/azure-quickstart-templates/blob/master/101-create-availability-set/azuredeploy.json